Add docs around setting up an app (#903)
tinacms/tinacms.org: content/docs/tina-cloud/dashboard.md

**Apps** connect Tina Cloud with a GitHub repository. An app is the **connection point between your site and your users**, allowing authorized users to access and modify a site's content.

### Connecting a GitHub repository

> To setup an app withiin the Tina Cloud dashboard, you must first authenticate with GitHub. A window will open asking you to give Tina.io access to your repositories. If you are not already logged in, your provider will prompt you for login credentials first. This authentication allows Tina Cloud to pull and push content to and from your GitHub respositiory.

Reviewer: ```suggestion
To set up an app within the Tina Cloud dashboard, you must first authenticate with GitHub. A window will open asking you to give Tina.io access to your repositories. If you are not already logged in, your provider will prompt you for login credentials first. This authentication allows Tina Cloud to pull and push content to and from your GitHub repository.
```

The next step is to choose the repository that houses your site's content. If you don't see your repository within the list of repositories on the dashboard, you may have to re-configure your Tina.io permissions within GitHub.

### App configuration

> On the last step of app creation, you will be prompted to enter a bit of configuration.

Reviewer: ```suggestion
On the last step of app creation, we ask you to enter a bit of configuration.
```

#### App Name

> This name will be shown to your users when they log in to the app.

Reviewer: ```suggestion
This name is shown to your users when they login to the app.
```

#### Site URL

> This field is used for security purposes so that users can only log-in through your site.

Reviewer: ```suggestion
This field is used for security purposes so that users can only log in through your site.
```

If you are developing locally, this value might be something like:

`http://localhost:3000`.

If Tina Cloud is configured on your production site, this value might be something like:

`https://<YOUR-SITE-NAME.com>`

**Only the URL origin is needed, so there is no need to include the path to any specific pages.**

## Using your app

> Once your app has been created, you will be able to use its Client ID within your site's Tina implementation. This will allow your users to log in to Tina Cloud from within your site, and start editing content. Learn more about the Tina Cloud client setup [here](http://localhost:3000/docs/tina-cloud/client/).

Reviewer: ```suggestion
Once your app is created, you can use its Client ID within your site's Tina implementation. This allows your users to log in to Tina Cloud from within your site, and start editing content. [Learn more about the Tina Cloud client setup](/docs/tina-cloud/client/).
```

Reviewer: Grammarly advises avoiding the future in technical writing.
